src/utilities/calculateApy/index.ts

```diff
@@ -0,0 +1,19 @@
+import BigNumber from 'bignumber.js';
+
+import { COMPOUND_DECIMALS } from 'constants/compoundMantissa';
+import { DAYS_PER_YEAR } from 'constants/daysPerYear';
+
+export interface CalculateApyInput {
+  dailyDistributedTokens: BigNumber;
+}
+
+const calculateApy = ({ dailyDistributedTokens }: CalculateApyInput) =>
```

Comment:
The param is not a number of tokens, but a daily percentage. Maybe the param name can generate some confusion

Reply:
Are you sure?
If you look at these lines: https://github.com/VenusProtocol/venus-protocol-interface/pull/1195/files#diff-9c59ea2aace70b47e0e69728f671a18140352c5b43e537c697349500de8dad5bR148-R154 the value we first calculate is `supplyDailyDistributedTokens`, which is one part of the value we display on the Market page (we add `borrowDailyDistributedTokens` to it ti get the total amount of tokens distributed per day): 
<img width="435" alt="Screen Shot 2023-07-21 at 13 23 13" src="https://github.com/VenusProtocol/venus-protocol-interface/assets/44675210/f773c48a-837e-41cc-88e1-2426065deb42">

This value is expressed in tokens, and it's this value we then pass to `calculateApy` to determine the distribution APY.